client/app/scripts/superdesk-templates/templates.js

```diff
@@ -156,8 +157,37 @@
                 desks.initialize()
                 .then(function() {
                     $scope.desks = desks.desks;
+
+                    /*
+                     * Returns desk name
+                     */
+                    $scope.getTemplateDesk = function getTemplateDesk(template) {
+                        return _.find($scope.desks._items , {_id: template.template_desk});
+                    };
+
+                    /*
+                     * Returns stage name
+                     */
+                    $scope.getTemplateStage = function getTemplateStage(template) {
+                        return _.find(desks.stages._items , {_id: template.template_stage});
+                    };
```

Comment:
They are defined inside promise because we need to wait for the desks to be initialize first, because we use that list to find current desk/stage...